src/orion/algo/space.py

```diff
@@ -484,15 +470,28 @@ def __contains__(self, point):
 
 
 class Integer(Real, _Discrete):
-    """Subclass of `Dimension` for representing integer parameters.
+    """Search space dimension that can take on any real value.
```

Comment:
> any real value.
What is the difference compared to the base class `Real`?

Reply:
Oopsie. Good catch, that was wrong copy paste from Real class.